Doc/library/string.rst

```diff
@@ -588,6 +589,14 @@ The available presentation types for :class:`float` and
    |         | as altered by the other format modifiers.                |
    +---------+----------------------------------------------------------+
 
+The available presentation types for :class:`complex` are same as for
+:class:`float`.  Both real and imaginary components of a complex number are
+formatted as floating-point numbers, accordingly specified presentaion type.
```

Comment:
Typo in "presentation". Also, I think this should be "are formatted as floating-point numbers, according to the specified presentation type."